components/x-audio/src/redux/player-logic.js

```diff
@@ -77,18 +85,19 @@ export const middleware = (store, audio = new Audio()) => {
 	audio.preload = 'metadata';
 
 	// debuging
-	[
-		'loadeddata',
-		'loadedmetadata',
-		'loadstart',
-		'progress',
-		'canplay',
-		'canplaythrough'
-	].forEach(evtName => {
-		audio.addEventListener(evtName, () => {
-			// console.log('audio eventy', evtName);
-		});
-	});
+	// [
+	// 	'loadeddata',
+	// 	'loadedmetadata',
+	// 	'loadstart',
+	// 	'progress',
+	// 	'canplay',
+	// 	'canplaythrough',
+	// 	'error'
+	// ].forEach(evtName => {
+	// 	audio.addEventListener(evtName, () => {
+	// 		console.log('audio eventy', evtName);
+	// 	});
+	// });
```

Comment:
Worth just killing this now?

Reply:
might be useful for when we get onto tracking